resty

```diff
@@ -168,6 +170,17 @@ for my $var (sort keys %ENV) {
     $env_list .= "env $var;\n";
 }
 
+my $main_include_directive = '';
+my $http_include_directive = '';
+
+if ($main_include) {
+    $main_include_directive = "include $main_include;"
```

Comment:
@nilnor Yeah, I agree. We can make it simple for now. But will you please make it possible to specify multiple instances of `--http-include` and `--main-include` options in a single command?